### DISCUSSION TITLE
No Update on Verification Status

### DISCUSSION THREAD

**Discussion by herrvonregen** [community/community#113421] [GitHub Education] [answered]
### Hi everyone,

I started my re-verification as a teacher again two weeks ago. However, I am not getting any feedback on the status. When can I expect approval?

**Comment by samus-aran** [comment]
Hi @herrvonregen 👋 

Having a look at your recent application it hasn't yet been submitted. You need to progress with your current application in order to be up for review. You can see your current application status [here](https://education.github.com/discount_requests/pack_application).

Let us know if you're encountering  any issue with submission. 👍

**Comment by herrvonregen** [comment]
Hi @samus-aran,
Thanks for your reply. I've had two problems. First, when I click on the provided link, I am redirected directly to the page where I could upload the proof and can not see my current status.
The second problem is that I've already tried to upload the image multiple times with different browsers. Every time I click "Process my Application," there is a short waiting cycle, and then the page reloads, and that's it. Is there maybe a problem with the submission page?

**Comment by herrvonregen** [comment, marked as answer]
Nevermind... I found the error. The file size was too big...

So, I got an enhancement proposal:
Please show a meaningful error message when things do not work during upload. (e.g., wrong file size)

**Reply by samus-aran** [reply]
Awesome to hear that you were successful 😄 We'll passed this feedback along to our internal team. 👍
